client/signup/steps/new-or-existing-site/index.tsx

```diff
@@ -101,7 +101,7 @@ export default function NewOrExistingSiteStep( props: Props ): React.ReactNode {
 		} else {
 			dispatch( removeSiteSlugDependency() );
 			dispatch( submitSignupStep( { stepName: 'difm-site-picker', wasSkipped: true } ) );
-			props.goToStep( 'site-info-collection' );
+			props.goToStep( 'difm-options' );
```

Comment:
Not a fan of this part, but a previous attempt at using branched steps did not work out correctly.

Reply:
@p-jackson do you have any insights here :)

Reply:
@p-jackson do you have any insights here :)

Reply:
I'll open a separate PR with the branched steps change. I'll add the link here when done!

Reply:
The issue with the previous approach was that I used the hook in a way that did not reset the excluded steps correctly. I have fixed it in https://github.com/Automattic/wp-calypso/pull/61946
Please have a look when you get the chance!

Reply:
Looks like you've figured it out :) I've added @arthur791004 on #61946 since he's definitely the branching guru